<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsClass</class>
 <widget class="QWidget" name="SettingsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>438</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>227</width>
    <height>231</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2270</width>
    <height>2310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PedestrianShape</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>391</width>
     <height>411</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Pedestrian Shape</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>201</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Shape</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>142</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Ellipse Resolution</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="CbEllipseResolution">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentIndex">
          <number>5</number>
         </property>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>201</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Color</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>178</width>
        <height>52</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="CbPedestrianGroup">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>First Group</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Second Group</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Third Group</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonChangePedestrianColor">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>change</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="chBpedestrianDefaultColor">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>174</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Default (read from file)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Trails/Captions</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>182</width>
       <height>104</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Number of Points</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="CbTrailPointsCount">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>40</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>70</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="CbTrailType">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Backward</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Symetric</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Forward</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Geometry</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="CbTrailGeometry">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Points</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Polygone</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Vertex width</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="CbTrailPolygoneWidth">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Trails</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>182</width>
       <height>104</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboCaptionColorMode">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Custom</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Orientation</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="ComboCaptionOrientation">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Custom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Billboard</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_19">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Automatic Rotation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="CbCaptionSize">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="maxCount">
         <number>200</number>
        </property>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>40</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>70</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>80</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>200</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkBoxCaptionAutoRotate">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Captions</string>
     </property>
    </widget>
    <widget class="QFrame" name="frameCaptionColorPreview">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>230</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 127);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Network</string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>53</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Host:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>53</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Locations">
    <attribute name="title">
     <string>Locations</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>219</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Directory in wich all generated screenshots and video will be saved. Default on the &lt;span style=&quot; font-weight:600;&quot;&gt;Desktop&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>output directory</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LEditOutputDir">
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Directory in wich all generated screenshots and video will be saved. Default on the &lt;span style=&quot; font-weight:600;&quot;&gt;Desktop&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="PBChangeOutputDir">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>163</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Change directory in wich all generated screenshots and video will be saved. Default on the &lt;span style=&quot; font-weight:600;&quot;&gt;Desktop&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Change</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>CbTrailPointsCount</sender>
   <signal>activated(QString)</signal>
   <receiver>SettingsClass</receiver>
   <slot>slotUpdateTrailSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CbTrailGeometry</sender>
   <signal>activated(QString)</signal>
   <receiver>SettingsClass</receiver>
   <slot>slotUpdateTrailSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CbTrailType</sender>
   <signal>activated(QString)</signal>
   <receiver>SettingsClass</receiver>
   <slot>slotUpdateTrailSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CbTrailPolygoneWidth</sender>
   <signal>activated(QString)</signal>
   <receiver>SettingsClass</receiver>
   <slot>slotChangeVerticesWidth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PBChangeOutputDir</sender>
   <signal>clicked()</signal>
   <receiver>SettingsClass</receiver>
   <slot>slotChangeOutputDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CbCaptionSize</sender>
   <signal>activated(QString)</signal>
   <receiver>SettingsClass</receiver>
   <slot>slotChangeCaptionSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboCaptionColorMode</sender>
   <signal>activated(QString)</signal>
   <receiver>SettingsClass</receiver>
   <slot>slotChangeCaptionColorMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ComboCaptionOrientation</sender>
   <signal>activated(QString)</signal>
   <receiver>SettingsClass</receiver>
   <slot>slotChangeCaptionOrientation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxCaptionAutoRotate</sender>
   <signal>clicked()</signal>
   <receiver>SettingsClass</receiver>
   <slot>slotChangeCaptionAutoRotation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CbEllipseResolution</sender>
   <signal>activated(QString)</signal>
   <receiver>SettingsClass</receiver>
   <slot>slotChangePedestrianShape()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotChangePedestrianShape()</slot>
  <slot>slotUpdateTrailSettings()</slot>
  <slot>slotPickPedestrianColor()</slot>
  <slot>slotChangeVerticesWidth()</slot>
  <slot>slotChangeOutputDir()</slot>
  <slot>slotChangeCaptionSize()</slot>
  <slot>slotChangeCaptionColorMode()</slot>
  <slot>slotChangeCaptionOrientation()</slot>
  <slot>slotChangeCaptionAutoRotation()</slot>
 </slots>
</ui>
